src/DevChatter.Bot.Core/Games/Hangman/HangmanGame.cs

```diff
@@ -116,7 +116,7 @@ public void AskAboutLetter(IChatClient chatClient, string letterToAsk, ChatUser
             }
 
             _guessedLetters.Add(new HangmanGuess(letterToAsk, chatUser));
-            if (Password.Contains(letterToAsk))
```

Comment:
If you do this on line 24 where the _password value is set, you don't have to do this `ToLower` with each check.

Reply:
Done.